src/openrct2-ui/scripting/CustomWindow.cpp

```diff
@@ -1205,7 +1205,7 @@ namespace OpenRCT2::Ui::Windows
                 {
                     if (customWidgetInfo->Name == name)
                     {
-                        return i;
+                        return static_cast<rct_widgetindex>(i);
```

Comment:
This change appears to be unrelated. Any particular reason?

Reply:
The solution wouldn't build for me in VS2019 preview, I quickly fixed it so I could get on with what I was doing and then forgot about it and ended up including those changes. I will separate it into a new commit or PR.